src/Components/ImageCommands/index.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { ClipboardCopy, Stack, StackItem } from '@patternfly/react-core';
+import { imageProps } from '../ProvisioningWizard/helpers';
+
+const launchInstanceCommand = (options) => {
+  return `gcloud compute instances create ${options.image_name}-instance --image-project ${options.project_id} --image ${options.image_name}`;
+};
+
+const ImageCommand = ({ image }) => {
```

Comment:
nit: The component's name is a bit ambiguous to me, maybe `GCPConsoleLaunch` or similar